SearchBar.js

```diff
@@ -1,15 +1,11 @@
-var NativeModules, PropTypes, RNSearchBar, React, SearchBar;
+import { Component, PropTypes } from 'react';
+import React, { NativeModules, requireNativeComponent } from 'react-native';
```

Comment:
Oops, my mistake, but you're right, the name ReactNative might be more appropriate. 👍